index.ts

```diff
@@ -26,7 +32,9 @@ module.exports = function resolve(resolves) {
 
         let keys = resolveKeys;
         if (config.optimizeDeps.include) {
-          keys = resolveKeys.filter(key => !config.optimizeDeps.include.includes(key));
+          keys = resolveKeys.filter(
+            key => !config.optimizeDeps?.include?.includes(key)
+          );
```

Comment:
`if (config.optimizeDeps.include) { }`  can ensure the security of the operate. There is no need to use the `?.` syntax.

Reply:
OK！I added a line of ts comments(// @ts-ignore) to solve the problem of ts becoming popular in this code segment